README.md

```diff
@@ -7,6 +7,13 @@ Complete interactive development program for Haskell
 Please see
 [the homepage for Intero for Emacs](http://commercialhaskell.github.io/intero).
 
+### Default key bindings
+
+- intero-goto-definition `M-.`
+- intero-info `C-c TAB`
```

Comment:
I think I'll tweak this to `C-c C-i` with `C-c TAB` in brackets. The `-i` is mnemonic for "info". I think TAB won't mean anything for remembering.